cartographer_grpc/map_builder_server.cc

```diff
@@ -95,6 +95,91 @@ void MapBuilderServer::MapBuilderContext::NotifyFinishTrajectory(
   map_builder_server_->NotifyFinishTrajectory(trajectory_id);
 }
 
+std::shared_ptr<cartographer::mapping_2d::Submap>
+MapBuilderServer::MapBuilderContext::GetSubmap2D(
```

Comment:
naming: this is not a getter. Submap2DFromProto? Read/Receive/UpdateSubmap2D?

Reply:
done